news/2022-08-15-15-anniversary-art-contest.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "15th Anniversary Art Contest"
+date: 2022-08-15 12:00:00 +0000
+---
+
+With osu!'s 15th anniversary approaching this September, it's time to take a look back at the past from the future with our latest art contest!
+
+![](https://assets.ppy.sh/contests/153/header.jpg)
```

Comment:
Shouldn't this be in the wikis cover folder

Reply:
no point uploading it twice really